std_msgs/Int32 maxFeatures
std_msgs/Int32 nOctave
std_msgs/Float64 contrastThresh
std_msgs/Float64 edgeThresh
std_msgs/Float64 sigma
